packages/eui/src-docs/src/views/steps/steps_example.js

```diff
@@ -231,7 +231,10 @@ export const StepsExample = {
           <strong>EuiSteps</strong> and <strong>EuiStep</strong>, the latter
           value will override the former. Additionally, the title size{' '}
           <EuiCode>xs</EuiCode> will automatically generate smaller steps
-          circles.
+          circles. <br />
+          The title size <EuiCode>xxs</EuiCode> affects only the size of the
+          accompanying <strong>unnumbered</strong> step indicator, but not the
+          title itself.
         </p>
```

Comment:
nit: We should probably use a `<></>` fragment for the parent and a new paragraph here
```suggestion
          circles.
        </p>
        <p>
          The title size <EuiCode>xxs</EuiCode> affects only the size of the
          accompanying <strong>unnumbered</strong> step indicator, but not the
          title itself.
        </p>
```